actions/websocket_actions.jsx

```diff
@@ -174,10 +174,22 @@ function handleEvent(msg) {
         handleUserUpdatedEvent(msg);
         break;
 
+    case SocketEvents.ROLE_ADDED:
```

Comment:
I thought this stuff was handled in redux?

Reply:
Yes, and no, redux have this handlers, but, the webapp use it's own implementation, it's expected they will converge in mattermost-redux, but currently, RN use mattermost-redux and webapp use it's own implementation.

Reply:
OK, great.